pilot/pkg/config/kube/gateway/testdata/eastwest-labelport.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: Gateway
+metadata:
+  name: eastwestgateway
+  namespace: istio-system
+  labels:
+    topology.istio.io/network: "network-1"
+    networking.istio.io/gatewayPort: 35443
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: istiod-grpc
+    port: 15012
+    protocol: TLS
+    tls:
+      mode: Passthrough
+  - name: istiod-webhook
```

Comment:
Do we still need to pass 15017 ? If I remember correctly, it was used for 'central istiod', but doesn't make sense for the 'istiod per cluster' model. I would either drop it or have a separate Gateway test for exposing central Istiod. 

I guess the same goes for 15012 - a remote gateway doesn't need to expose the remote Istiod. 

I know this is a test - but will confuse readers and may accidentally be used this way.

Reply:
You're probably right. This is mostly copypaste. I'll remove it.